packages/buckets/src/buckets.spec.ts

```diff
@@ -19,9 +21,18 @@ describe('Buckets...', () => {
   let buck: CreateObject
   let fileSize: number
 
+  let dev: SignupResponse.AsObject
+  const apiKeyInfo = { key: '' }
+
   before(async () => {
```

Comment:
Note for future reference, we shouldn't be using arrow functions for async functions with mocha anymore (I'm also guilty of this), the behavior is often unexpected, because we end up with the wrong "this" context. Any `before`, `after`, `beforeEach`, `afterEach`, and `it`'s should all be `async function () {...}`, whereas `describe` and `context` can be arrow functions (but should probably just be `async function` to be consistent and safe.

Reply:
Looks like you're already doing this with new tests, so kudos!

Reply:
okay, went ahead and took care of the others while in there